posthog/models/filters/mixins/property.py

```diff
@@ -39,7 +42,16 @@ def property_groups(self) -> PropertyGroup:
         else:
             loaded_props = _props
 
-        return self._parse_property_group(loaded_props)
+        # if grouped properties
+        if isinstance(loaded_props, dict) and "type" in loaded_props and "groups" in loaded_props:
+            try:
+                return self._parse_property_group(loaded_props)
+            except Exception as e:
+                # TODO: make exception class precise?
```

Comment:
This is IMO a requirement for merging this. - we should not be catching generic exception ever.